internal/runbits/dependencies/summary.go

```diff
@@ -0,0 +1,57 @@
+package dependencies
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/pkg/platform/runtime/artifact"
+	"github.com/ActiveState/cli/pkg/platform/runtime/buildplan"
+)
+
+// OutputSummary lists the given runtime dependencies being installed along with their
+// subdependencies (if any).
+func OutputSummary(out output.Outputer, directDependencies []artifact.ArtifactID, artifacts artifact.Map) {
+	if len(directDependencies) == 0 {
+		return
+	}
+
+	dependencies := make(map[artifact.ArtifactID][]artifact.ArtifactID)
+	for _, artifactId := range directDependencies {
+		subdependencies := buildplan.RecursiveDependenciesFor(artifactId, artifacts)
```

Comment:
Currently, this shows top level dependencies, whereas the story calls for "direct requirements of the project". I worded that very poorly though.. what I meant was it should show the requirements the user specified in their commit.

Reply:
Okay. I've updated this and updated the sample output in the PR summary.